pkg/monitors/kubernetes/volumes/volumes.go

```diff
@@ -110,6 +110,15 @@ func (m *Monitor) getVolumeMetrics() ([]*datapoint.Datapoint, error) {
 			if v.CapacityBytes != nil {
 				dps = append(dps, sfxclient.Gauge("kubernetes.volume_capacity_bytes", dims, int64(*v.CapacityBytes)))
 			}
+			if v.Inodes != nil {
+				dps = append(dps, sfxclient.Gauge("kubernetes.volume_inodes", dims, int64(*v.Inodes)))
```

Comment:
Use the constants in genmetadata.go (e.g. `kubernetesVolumeInodes`) instead of string literals.  Also replace the existing metrics with constants for consistency.